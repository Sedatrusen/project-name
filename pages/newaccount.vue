<template>
  <div class="sayfa">
    <div class="card">
      <img class="logo" src="../assets/blizzlogo.png" alt="" />
      <h1>Get starded</h1>
      <p>
        Let us verify some information about you to help set up your
        account.<router-link to="/">Learn why we need this. </router-link>
      </p>
      <select v-model="selected">
        <option>Turkey</option>
        <option>USA</option>
        <option>England</option>
      </select>
      <div>
        <input
          v-model="message"
          class="tarih"
          placeholder="Date of Birth(dd/mm/yyyy)"
        />
      </div>
      <button @click="greet">Continue</button>
      <p class="yazi">OR SING UP WITH</p>
      <div class="emailout">
        <button class="sosyal" type="submit">
          <img src="../assets/facebookimg.png" alt />
        </button>
        <button class="sosyal" type="submit">
          <img src="../assets/googleimg.png" alt />
        </button>
        <button class="sosyal" type="submit">
          <img class="img" src="../assets/applelogo.png" alt />
        </button>
        <p>Already have an account? <nuxt-link to="login">Login</nuxt-link></p>
      </div>
    </div>
  </div>
</template>

<script scoped>
export default {
  layout: 'accountly',
  data() {
    return { selected: 'Turkey' }
  },
}
</script>

<style scoped>
.sayfa {
  display: grid;
  background: no-repeat;
  background-size: cover;
  background-image: url(../assets/accountarkaplan.jpg);
  grid-template-columns: 2fr 2fr 2fr;
  width: 100%;
  height: 100%;
}
.card {
  background-color: rgb(255, 255, 255, 0.1);
  max-width: 500px;
  margin: 50px;
  grid-column: 2/3;
  border: 1px solid rgba(255, 255, 255, 0.15);
  display: flex;
  justify-items: center;
  align-items: center;
  padding: 2rem;
  text-align: center;
}
.logo {
  height: 60px;
  width: 120px;
}
select {
  width: 400px;
  height: 40px;
  color: white;
  background-color: black;
}
p {
  color: white;
}
.tarih {
  width: 395px;
  height: 40px;
  color: white;
  background-color: black;
  border: none;
  margin-top: 10px;
}
button {
  width: 400px;
  margin-top: 10px;
  height: 40px;
  font-size: 17px;
  text-align: center;
  background-color: #0e86ca;
  color: #fff;
  border: 1px solid #10a1de;
  border-radius: 0;
  transition: background-color 0.2s, box-shadow 0.2s, background-color 0.2s,
    border-color 0.2s, color 0.2s;
  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.3),
    0 0 0 1px rgba(255, 255, 255, 0.15) inset;
}
.sosyal {
  margin-left: 10px;
  background: white;
  line-height: 20px;
  width: 44px;
  height: 44px;
}
.sosyal:first-child {
  background-color: #165dee;
}
.img {
  width: 20px;
  align-self: center;
}
a {
  color: #139cff;
}
h1 {
  color: #fff;
  font-weight: lighter;
}
</style>
