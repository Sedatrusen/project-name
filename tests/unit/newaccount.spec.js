import { shallowMount, createLocalVue } from '@vue/test-utils'
import Vuex from 'vuex'
import newaccount from '@/pages/newaccount.vue'

const localVue = createLocalVue()

localVue.use(Vuex)

describe('newaccount component', () => {
  let actions
  let store
  let getters
  let state
  beforeEach(() => {
    state = { products: { Products: [{ id: '1' }, { id: '2' }] } }
    actions = {
      addproducts: jest.fn(),
      addoptions: jest.fn(),
      chechwhislist: jest.fn(),
    }
    getters = {
      whislists: () => 2,
      cartTotal: () => 'Total : â‚¬',
    }
    store = new Vuex.Store({
      actions,
      getters,
      state,
    })
  })

  it('newaccount component set etme', () => {
    const wrapper = shallowMount(newaccount, {
      computed: {
        blizzardgames: () => 'value_2',
        partnergames: () => '2',
      },

      methods: { setItems: () => {} },
      localVue,
      store,
    })
    expect(wrapper.exists()).toBe(true)
    expect(wrapper.html).toMatchSnapshot()
  })
})
