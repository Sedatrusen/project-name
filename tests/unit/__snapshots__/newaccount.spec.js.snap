// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`newaccount component newaccount component set etme 1`] = `[Function]`;
